tasks/monitoring.yml

```diff
@@ -24,3 +25,25 @@
     name: "{{ monitoring_package_name }}"
     state: "{{ package_state }}"
 #TODO(rmoriar1): Clean up the repo when package_state is absent.
+
+- name: Copy main config file onto the remote machine
+  copy:
+    src: "{{ monitoring_main_config_file }}"
+    dest: /etc/stackdriver/collectd.conf
+    force: yes
+    mode: 0644
+    validate: "{{ monitoring_binary }} -tC %s"
+  when: monitoring_main_config_file|length > 0
```

Comment:
Shouldn't we provide a default? Are you just relying on the package to provide a config file in that case?

Reply:
Yes, if `monitoring_main_config_file` is unspecified I'm assuming they will use the config file that comes w/ the installation.